sample/library/build.gradle

```diff
@@ -67,5 +67,6 @@ if (project.rootProject.name != 'buildSrc') {
     artist {
         interfaceClassName = "com.uber.artist.mylibrary.MyView"
         viewNamePrefix = "My"
+        generateKotlin = false
```

Comment:
Did you mean true? Or just demonstrating even though the default is false?

Reply:
Was just demo-ing